scripts/build.py

```diff
@@ -28,7 +28,7 @@
 YUICOMPRESSOR_DIR = os.path.join(
     '..', 'oppia_tools', 'yuicompressor-2.4.8', 'yuicompressor-2.4.8.jar')
 
-FILE_EXTENSIONS_TO_COPY_WITHOUT_MINIFICATION = ['.json', '.png']
```

Comment:
Aren't JSON and PNG files going to be copied with minification after this change?